### ISSUE TITLE
Model training on offline machine

### ISSUE THREAD

**Issue by icaroryan** [Deci-AI/super-gradients#1880]
### 💡 Your Question

I'm training my models on offline clusters. Can I save a model with its weights locally to avoid repeatedly downloading them from the internet? Downloading isn't possible in my offline environment. I can't use `models.get(...)` because it needs to download the weights each time.

**Comment by BloodAxe** [comment]
Yes, it is possible, the only change you need to do is to modify `models.get(..., pretrained_weights=....` line to `models.get(..., checkpoint_path=ABSOLUTE_PATH_TO_SAVED_CHECKPOINT, checkpoint_num_classes=80_IF_YOU_ARE_USING_COCO_PRETRAINED_WEIGHTS)`. This will load a local checkpoint instead of the remote file.
By the way, actual download happens only once and then file is stored locally in a filesystem cache in the users directory.

**Comment by icaroryan** [comment]
Wonderful, I'll give it a try! Thank you, Eugene
